### ISSUE TITLE
Cant deploy api on a separate BG(business group)

### ISSUE THREAD

**Issue by dinglezos-2022** [mulesoft/docs-code-builder#191]
I have a root BG and a child BG, but there is no option to choose the child BG, I can only deploy to the root BG.

**Comment by macrocks** [comment]
@dinglezos-2022 Are you trying to deploy using Anypoint CLI, CI.CD pipline or maven plugin or Anypoint Platform UI option? 
can you elaborate or povide steps which you are trying to perform?

**Comment by dinglezos-2022** [comment]
@macrocks None of that. I used the Anypoint Code Builder.

PS> I just realized, I didn't mention the method by which I was trying to deploy, sorry 😒

**Comment by macrocks** [comment]
@dinglezos-2022  are you trying to deploy using ACB installed on your machine or from anypoint platform ACB? also are you logged to ACB using root BG user or child BG user?

**Comment by dinglezos-2022** [comment]
@macrocks From Anypoint Platform. I use the root BG.

**Comment by macrocks** [comment]
@dinglezos-2022   at this moment that feature is neither available on anypoint code builder nor anypoint cli or mule maven plugin.  I guess there is specific purpose for business group i.e. seperation of concerns and adding this feature may violate it.

**Comment by dinglezos-2022** [comment]
@macrocks That doesn't make sense to me. 
If I have access to deploy from Anypoint Studio into said BG, then I should be able to deploy from anywhere, including Code Builder. 

Also, are you sure we can't deploy using Maven ? 
I thought you can define which BG you want to deploy, and as long you have the permissions you can?

**Comment by macrocks** [comment]
@dinglezos-2022  you are correct. we can deploy on any BG by selecting option during deploy option in anypoint studio  as well as for maven plaugin you can pass BG as parameterized value.

For ACB, looks like you login first then we can do deployment due to which it doesn't provide option where to deploy, this feature needs to requested so Mulesoft team can work on it. 
@dukesphere  - can this be raised on ACB's feature backlog?

**Comment by Crispy-Salesforce** [comment]
Hi, @macrocks!
If you find this issue in the latest version of Anypoint Code Builder, please, contact the support team so they can file a bug.
Thanks for your collaboration and support!
